COCOAPODS.md

```diff
@@ -1,150 +1,343 @@
 # CocoaPods integration
 
-Starting with 1.3.30, an experimental integration with [CocoaPods](https://cocoapods.org/) is added
-to Kotlin/Native. This feature allows you to represent a Kotlin/Native Gradle-project as a
-CocoaPods dependency. Such a representation provides the following advantages:
+Kotlin/Native provides integration with the [CocoaPods dependency manager](https://cocoapods.org/). You can add
+dependencies on Pod libraries stored in the CocoaPods repository or locally as well as use a Kotlin/Native Gradle
+project as a CocoaPods dependency (Kotlin Pod).
 
- - Such a dependency can be included in a Podfile of an Xcode project and automatically built (and rebuilt)
- along with this project. As a result, importing to Xcode is simplified since there is no need to
- write corresponding Gradle tasks and Xcode build steps manually.
+You can manage Pod dependencies right in Intellij IDEA enjoying its benefits of working with code, such as code
+ highlighting and completion. You can build the whole Kotlin project with Gradle as a whole not switching to Xcode. 
+ Go to Xcode only when you need to write Swift/Objective-C code or run your application on a simulator or device.
  
- - When building from Xcode, you can use CocoaPods libraries without writing
- .def files manually and setting cinterop tool parameters. In this case, all required parameters can be
- obtained from the Xcode project configured by CocoaPods.
+Depending on your purpose, you can add dependencies between:
+* [A Kotlin project and a Pod library from the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository)
+* [A Kotlin project and an Objective-C Pod library stored locally](#add-a-dependency-on-an-objective-c-pod-library-stored-locally)
```

Comment:
How "Objective-C Pod library" here is different from "Pod library" above?

Reply:
I think that the difference in storage place: Cocoapods repo vs user's filesystem. In both cases only Objective-C libraries could be used.

Reply:
>  In both cases only Objective-C libraries could be used.

Then we should stick to the same wording here. Otherwise this can be understood as "published Swift pods are supported, local ones aren't", which is not true.

Reply:
Agree :). Will remove Objective-C and use Pod library everywhere

Reply:
Done